packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.targets

```diff
@@ -47,4 +36,27 @@
 
   </Target>
 
+  <!-- 
+    This warning can be suppressed by setting $(MSBuildWarningsAsMessages) property, like this:
+    <PropertyGroup>
+      <MSBuildWarningsAsMessages>$(MSBuildWarningsAsMessages);NETSDK1106</MSBuildWarningsAsMessages>
+    </PropertyGroup>
+  --> 
+  <Target Name="_WindowsDesktopFrameworkRequiresUseWpfOrUseWindowsForms"
```

Comment:
Should we condition this check on `"'$(_TargetFrameworkVersionWithoutV)' >= '3.0'"` ? That way, when a `Microsoft.NET.Sdk.WindowsDesktop` app targets `netcorapp2.2`, it will get `WindowsDesktopFrameworkRequiresVersion30` warning, but not `WindowsDesktopFrameworkRequiresUseWpfOrUseWindowsForms`